### ISSUE TITLE
Efficiency Enchant Cost Too High for Flux Bore

### ISSUE THREAD

**Issue by AENykorak** [AllTheMods/ATM-6#1536]
AllTheMods6-1.6.1
Efficiency when attempting to apply to the Thermal Series Flux Bore has an enchantment cost of 128 or greater.
The enchantment cost of 128 occurs with Efficiency I and increases from there.

**To Reproduce**
1. Attempt to apply Efficiency I to the Thermal Series Flux Bore in an anvil

**Screenshots**
![image](https://user-images.githubusercontent.com/64785727/116395946-becd9380-a7f2-11eb-87aa-441d644c9797.png)

**Additional context**
Normal enchanting cost applies to regular items and flux saw the flux saw when tested.

**Comment by AENykorak** [comment]
The same issue occurred with Silk Touch.

![image](https://user-images.githubusercontent.com/64785727/116396372-4fa46f00-a7f3-11eb-9e55-b395641b7222.png)

**Comment by JosieGr** [comment]
Had this issue also with just a vanilla shield with many enchantments. It would say it required incredibly high levels to repair but only take, say, 4 or so. And it go to the point it wanted 100+ levels I didn't have even though a repair required in reality much less.

**Comment by AENykorak** [comment]
I have discovered the problem with the enchantment bug.
I believe it has to do with the Cyclic Disenchanter. It strips the enchantments off the item but does not remove the actual enchanting level.

I had the issue happen with a Silent Gear crafted Unobtanium sword, when I stripped it down to put new enchants on it required 75 levels for a basic Indestructible. In an attempt to troubleshoot I tried the Industrial Foregoing Enchantment Remover to remove the enchantment. It reset the enchantment level back to 0 once all enchantments were stripped and the indestructible cost the normal 2 levels.

I would recommend not using the Cyclic Disenchanter until it is removed or fixed.

The creators of the mod should also be notified.
